hbase-shaded/hbase-shaded-mapreduce/pom.xml

```diff
@@ -168,230 +209,4 @@
         </dependency>
     </dependencies>
 
-    <profiles>
```

Comment:
these were moved to hbase-shaded-mapreduce-provided-dependencies so that they actually get included in the dependency reduced pom (since we use promoteTransitiveDependencies)